### PR TITLE
Private/devidask/update readme

### PR DESCRIPTION
**Pull request by devidask27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Usage:
   pf9ctl [command]
   
 Available Commands:
+  attach-node Attaches node to kubernetes cluster
   bundle      Gathers support bundle and uploads to S3
   check-node  Check prerequisites for k8s
```
Comment (nikhilbandi) [inline README.md:38]: Replace k8s with kubernetes

File: README.md [hunk 2]
```diff
@@ -90,6 +91,8 @@ Use "pf9ctl config [command] --help" for more information about a command.
         - Ports Check
         - OS Check
         - Existing Kubernetes Cluster Check
+        - Check exec permission on /tmp
```
Comment (nikhilbandi) [inline README.md:94]: Should be: Check execute permissions on /tmp folder

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
Please do minor corrections.

File: README.md (hunk 2 above)
Comment (devidask27) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (devidask27) [reply README.md:38]: Done

**Review by nikhilbandi — APPROVED** [review]
Looks good.

**Pull request merged**